### PR TITLE
UIMARCAUTH-359 Don't validate hrid start number for FOLIO source files (follow-up)

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Don't validate hrid start number for FOLIO source files because they don't have this field

## Issues
[UIMARCAUTH-359](https://folio-org.atlassian.net/browse/UIMARCAUTH-359)

### CODE REVIEW

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: src/settings/ManageAuthoritySourceFiles/getValidators.js [hunk 1]
```diff
@@ -50,7 +53,11 @@ const validators = {
 
     return undefined;
   },
-  [authorityFilesColumns.START_NUMBER]: function validateStartNumber({ hridManagement }) {
+  [authorityFilesColumns.START_NUMBER]: function validateStartNumber({ hridManagement, source }) {
+    if (source === SOURCES.FOLIO) {
+      return undefined;
+    }
```
Comment (Dmytro-Melnyshyn) [inline src/settings/ManageAuthoritySourceFiles/getValidators.js:59]: Maybe we should do the same for the other fields.

Comment (BogdanDenis) [reply src/settings/ManageAuthoritySourceFiles/getValidators.js:59]: Other fields in FOLIO records are filled by default, so I think it's ok

**Review by Terala-Priyanka — APPROVED** [review]
Please update changelog.md if it applies here.

**Comment by BogdanDenis** [comment]
Hey @Terala-Priyanka, this is a follow-up request, so the CHANGELOg entry is already there

**Pull request merged**